catalog/imaging/imaging-transformer-nitf/src/main/java/org/codice/alliance/transformer/nitf/gmti/IndexedMtirpbAttribute.java

```diff
@@ -24,30 +24,10 @@
 import ddf.catalog.data.impl.BasicTypes;
 
 enum IndexedMtirpbAttribute implements NitfAttribute<TreGroup> {
-    INDEXED_TARGET_CLASSIFICATION_CATEGORY("targetClassificationCategory",
-            "TGT_CAT",
-            tre -> getClassificationCategory(tre)),
-    INDEXED_TARGET_AMPLITUDE("targetAmplitude",
-            "TGT_AMPLITUDE",
-            tre -> GmtiTreUtility.getTreValue(tre, "TGT_AMPLITUDE")),
-    INDEXED_TARGET_HEADING("targetHeading",
-            "TGT_HEADING",
-            tre -> GmtiTreUtility.getTreValue(tre, "TGT_HEADING")),
-    INDEXED_TARGET_GROUND_SPEED("targetGroundSpeed",
-            "TGT_SPEED",
-            tre -> GmtiTreUtility.getTreValue(tre, "TGT_SPEED")),
-    INDEXED_TARGET_RADIAL_VELOCITY("targetRadialVelocity",
-            "TGT_VEL_R",
-            tre -> GmtiTreUtility.getTreValue(tre, "TGT_VEL_R")),
-    INDEXED_TARGET_LOCATION_ACCURACY("targetLocationAccuracy",
-            "TGT_LOC_ACCY",
-            tre -> GmtiTreUtility.getTreValue(tre, "TGT_LOC_ACCY")),
     INDEXED_TARGET_LOCATION("targetLocation",
```

Comment:
Is there a normalized taxonomy attribute for this? I would expect all of these strings to be populated by constants.